app/helpers/dcv/catalog_helper_behavior.rb

```diff
@@ -100,4 +100,26 @@ def button_to_next_document(next_document, opts={})
   def pcdm_file_genre_display value
     t("pcdm.file_genre.#{value}")
   end
+
+  def total_dcv_asset_count
+    Rails.cache.fetch('total_dcv_asset_count', expires_in: 12.hours) do
+      solr_params = {
+        qt: 'search',
+        rows: 0,
+        fq: ["active_fedora_model_ssi:GenericResource"],
+        facet: false
+      }
+      response = Blacklight.solr.get 'select', :params => solr_params
+      response['response']['numFound'].to_i
+    end
+  end
+
+  def rounded_down_and_formatted_dcv_asset_count
+    round_to_nearest = 5000 # e.g. round 12,345 down to nearest 5000: 10,000
+    exact_total = total_dcv_asset_count
+    return exact_total if exact_total < round_to_nearest
+
+    count_to_return = total_dcv_asset_count / round_to_nearest * round_to_nearest
```

Comment:
Using `exact_total` instead of `total_dcv_asset_count` would make one less solr query in some cases.

Reply:
Ah, thank you!  I actually meant to do that.  That was a copy and paste error.